athena/templates/partition_tdr_flowlogs_mgmt_jenkins.sql.tpl

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE tdr_flowlogs_mgmt_jenkins
+ADD PARTITION (`date` = '2020-06-04')
```

Comment:
Should this date be hardcoded?

Reply:
these are just examples - ideally, there would be an automated script creating a new partition every day

Reply:
ok